src/status_im/commands/core.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.commands.core
+  (:require
+   [re-frame.core :as re-frame]
+   [status-im.ethereum.json-rpc :as json-rpc]
+   [status-im.utils.fx :as fx]))
+
+(fx/defn handle-prepare-accept-request-address-for-transaction
+  {:events [:commands/prepare-accept-request-address-for-transaction]}
```

Comment:
use `::` and then alias the events `::commands` where they are used so that you have explicit usage of that namespace and there is no need to add the require in `events` which could get accidentally refactored away